### ISSUE TITLE
Impact of VIB on Overfitting

### ISSUE THREAD

**Issue by neerajkuhike** [rabeehk/vibert#7]
Hi ,

what is the setup for this experiment? Are you fixing the beta value and doing the experiment and checking the loss and doing same for other beta values OR using KL annealing? In the experiment, we have two losses CE and KL loss. Are you just seeing the CE loss to draw the plot ?

**Comment by rabeehk** [comment]
Dear @neerajkuhike 
first please accept my apologies for missing this issue, please always feel free to also send me an email at rabeeh.k68@gmail.com or rkarimi@idiap.ch 

In Figure 3, we fix K, which is the bottleneck size, and we are only changing the beta, and then plot the cross entropy loss on the validation set of each dataset. So we do not consider KL-loss for this plot. This is similar to the plot done in the original work of Alemi et al.

**Comment by ShellingFord221** [comment]
Hi, is there a relationship between bottleneck size and the number of pseudo-inputs in [VampPrior?](https://arxiv.org/pdf/1705.07120.pdf) Somehow I think they are just related... Thanks!

**Comment by rabeehk** [comment]
Hi
I am not aware of this work to be able to comment on this. Sorry.
